script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -70,13 +70,13 @@ But in order to talk about Ancestors, we need to lay some groundwork first..."
 [sha:494686c24351bd92c15d3e5d3dfe8459c2f29935]{
 -- Page 580, Offset 18026.
 -- 「じゃーん！
-"Ta-da!
+"Ta-da! //Currently don't have access to Yuzu but does the white board appear here or on line 84 or here? If it's on line 84 should this line be something like "Then!" or "Th-en!" to flow better with the follwing line?
 }
 [sha:ffb1c8db94c3ccbef2c675588adaa8d9db53b7be]{
 -- Page 580, Offset 18027.
 -- はい、ここで吸血鬼講座・上級編を開始しまーす！」
 // Course name format matched to those in 07_00_ARC07_3_1_1 and D4
-Without further ado, let's begin our course on Advanced Vampire Studies!"
+Without further ado, let's begin our Advanced Vampire Studies course!"
```

Comment:
Original reads more naturally to me, at least when I substitute in 'real' courses, but I can't really find any definitive answer
> let's begin our course on molecular biology!
> let's begin our molecular biology course!

Reply:
I changed order because of the addition "Studies" at the end, since in my brain if this were substituted in like a real course it would be like.

"let's begin our course on Advanced Vampires!" but might just be me.

Reply:
hmm, yeah you might be right. 
> begin our advanced social studies course 

does seem more correct than
> begin our course on advanced social studies

Reply:
Guess we'll wait for a third opinion for now since I'm not exactly sure either tbh.

Reply:
"let's begin our course on Advanced Vampires!" is my preferred version

Reply:
Hm, "course on advanced vampires" suggests that it is the vampires are advanced, not the level of the course though.

Reply:
Now that I think about it, should it be "Vampires Studies Advanced"/"Vampires Advanced course"? Since the other option had it as "Vapire fundamentals" right? So then is it talking about the course being of higher level or that the course is about the big boy vampires?

Reply:
I believe it is an (advanced course) on vampires, not a course on (advanced vampires)